docs/index.md

```diff
@@ -50,13 +50,13 @@ provider "redpanda" {
 ```terraform
 provider "redpanda" {}
 
-resource "redpanda_namespace" "test" {
-  name = var.namespace_name
+resource "redpanda_resourcegroup" "test" {
```

Comment:
The name convention doesn't seem to be consistent, shouldn't it be `redpanda_resource_group` instead? 

See for example Azure's version of the resource: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_group.html

Reply:
I intentionally used `resourcegroup` instead of `resource_group` to follow AWS provider convention:
- https://registry.terraform.io/providers/hashicorp/aws/latest/docs (autoscaling, bcmdataexport, etc...)

It leaves us chance to extend the resourcegroup functionality into other types of resources, for example: `redpanda_resourcegroup_roles`

Why do you mean not consistent? which other resource in this provider has a compound name?

Reply:
I can change it if we have other instances in our codebase where we favor `resource_group` so it's consistent across different uses in the Redpanda Cloud tooling. Not opposed to that.

Reply:
What I mean with consistency is that we are using _ to separe words in the names of all the other resources but resource_group, which is 2 words.

Reply:
We have:
- redpanda_resourcegroup
- redpanda_network
- redpanda_cluster
- redpanda_user
- redpanda_acl
- redpanda_topic

This is the first case of a compound name.

Reply:
> It leaves us chance to extend the resourcegroup functionality into other types of resources, for example: redpanda_resourcegroup_policy

I'm not sure I follow, why can't it be extended as `redpanda_resource_group_policy`?

Reply:
The AWS provider is not the best example of consistent conventions. See for example: [aws_api_gateway_account](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_account) and [aws_apigatewayv2_api](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/apigatewayv2_api)

Reply:
It looks like their convention nowadays is `aws_[compounded product name]_[resource_name]`, probably because of how vast their API is.

Reply:
Solved, changed to resource_group.